tests/test_series.py

```diff
@@ -445,7 +445,11 @@ def test_types_groupby_level() -> None:
         [(0, 0, 1), (0, 1, 2), (0, 0, 3)], names=["col1", "col2", "col3"]
     )
     s = pd.Series([1, 2, 3], index=index)
-    s.groupby(level=["col1", "col2"]).sum()
+    check(
+        assert_type(s.groupby(level=["col1", "col2"]).sum(), "pd.Series[int]"),
```

Comment:
You do need the quotes here around `pd.Series[int]`, because the generic version is not known at runtime.